### ISSUE TITLE
Autofill email list window stuck on the login flow screens

### ISSUE THREAD

**Issue by mzorz** [wordpress-mobile/WordPress-Android#6855]
This behavior was originally found while testing https://github.com/wordpress-mobile/WordPress-Android/pull/6837 with @theck13 

### Expected behavior
When Google autofill appears on a text field, it should go away if the user navigates to other Activities.

### Actual behavior
The Google autofill window gets stuck on the same position, even when changing

### Steps to reproduce the behavior
1. on a device with multiple google accounts setup, clean the app, and make sure you have Google Autofill set to ON (Settings, System, Language & Input > Advanced, Auto fill, turn on)
2. on the main welcome screen, tap on LOGIN. You should be taken to the screen where your email address needs to be entered. This is where auto fill should do its work, and a small list with your email addresses should appear:
<img width="180" alt="screen shot 2017-11-13 at 12 50 52 pm" src="https://user-images.githubusercontent.com/6597771/32735107-8a8bfcda-c89c-11e7-8420-a21f1f86f72f.png">

3. on the login screen, tap on "Login with Google"
4. once the account chooser is shown, tap BACK, maybe a couple of times to get to the first welcome screen again.
5. you should see the list is stuck and doesn't go away
<img width="184" alt="screen shot 2017-11-13 at 12 50 01 pm" src="https://user-images.githubusercontent.com/6597771/32735263-e7a82628-c89c-11e7-9b12-e9e53af67aac.png">

**Comment by theck13** [comment]
The app also crashed when long-pressing the password field on the email/password screen and choosing Autofill.  The stack trace is below.

```
11-17 16:20:03.283 3055-3055/org.wordpress.android.beta E/AndroidRuntime: FATAL EXCEPTION: main
    Process: org.wordpress.android.beta, PID: 3055
    java.lang.NullPointerException: Attempt to invoke virtual method 'void android.view.View.getBoundsOnScreen(android.graphics.Rect)' on a null object reference
        at android.app.assist.AssistStructure$WindowNode.<init>(AssistStructure.java:484)
        at android.app.assist.AssistStructure.<init>(AssistStructure.java:1908)
        at android.app.ActivityThread.handleRequestAssistContextExtras(ActivityThread.java:3035)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1807)
        at android.os.Handler.dispatchMessage(Handler.java:105)
        at android.os.Looper.loop(Looper.java:164)
        at android.app.ActivityThread.main(ActivityThread.java:6541)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:240)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:767)
```

**Comment by mzorz** [comment]
closed via #6880  - should be revisited once Google fixes the problem as #6880 effectively prevents Autofill from working on the login forms
